### ISSUE TITLE
Code smell not detected

### ISSUE THREAD

**Issue by fadc80** [felipebz/sonar-plsql#9]
I'm trying to test this plugin with the following source code:

````
-- dead-code.sql
begin
  raise my_error;
  log('finished'); -- esse código nunca será executado
end;
````
This code was extracted from one of the plugin's rules and is classified as code smell, but Sonar does not detect it. Sonar scanner reports que following warn message:  

````
WARN: Missing blame information for the following files:
WARN:   * /sonar-examples/projects/languages/plsql/plsql-sonar-runner/src/dead-code.sql
````

This test was ran with...

1. sonarqube 6.0
2. sonar-scanner 2.6.1

**Comment by felipebz** [comment]
Hi. Thanks for the interest in my plugin.

I'm sure that the "Missing blame information" is not the cause of the problem. The SonarQube Scanner show this message when there are uncommited files and this doesn't affect the analysis.

Can you confirm if the plsql:DeadCode rule is enabled in your quality profile? Please check at http://YOUR_SONAR_SERVER/coding_rules#rule_key=plsql%3ADeadCode

It should be like:

![image](https://cloud.githubusercontent.com/assets/13829/18612379/a5c60a96-7d2e-11e6-9723-9d4a79f81d5e.png)

**Comment by fadc80** [comment]
You're right! The rules inside Sonar way PL/SQL quality profile were deactivated. This is my first touch with SonarQube and I didn't know rules are not enabled by default after installing a new plugin. Is this really the expected behaviour?

**Comment by felipebz** [comment]
No, it's not the expected behavior. I've fixed this bug in the [last snapshot](https://ci.appveyor.com/project/felipebz/sonar-plsql/build/artifacts). Thanks!

If you want, after the update you can restore the "Sonar way" profile using the "Quality Profiles" page.

**Comment by felipebz** [comment]
Are you using the same database, right?

Unfortunately, SonarQube doesn't update the existing profiles when the plugin is updated. So you need to enable the rules manually or restore the built-in profile in the Quality Profiles page (http://SERVER/profiles).

![image](https://cloud.githubusercontent.com/assets/13829/18636202/4ea9f246-7e5e-11e6-827c-8a429217050d.png)

![image](https://cloud.githubusercontent.com/assets/13829/18636181/3ba18b32-7e5e-11e6-9ce2-f1daefd188d8.png)

**Comment by fadc80** [comment]
Yes, it is the same. Sorry, now I got it. Thanks! Issue resolved.

**Comment by fadc80** [comment]
Yes, it is the same. Sorry, now I got it. Thanks! Issue resolved.
